src/components/chat/comment-card.tsx

```diff
@@ -91,9 +91,9 @@ export const CommentCard: React.FC<IProps> = ({ activeNavTab, user, postedCommen
                   }
                 </div>
                 {
-                  appConfig.showCommentTag && comment.tag &&
+                  appConfig.showCommentTag && comment.tags &&
                   <div className="comment-dropdown-tag">
-                    { comment.tag }
+                    { comment.tags }
```

Comment:
Since `comment.tags` is of type `string[]`, should we do `comment.tags && comment.tags[0] ` ?

Reply:
It still works atm, but I feel like that might be best practice in the case we have multiple tags.

Reply:
I would suggest `coment.tags.join(", ")`.

Reply:
Kirk's suggestion seems like a good solution.